erpnext/accounts/report/sales_payment_summary/sales_payment_summary.js

```diff
@@ -35,6 +35,11 @@ frappe.query_reports["Sales Payment Summary"] = {
 			"fieldname":"is_pos",
 			"label": __("POS?"),
 			"fieldtype": "Check"
-		}
+		},
+		{
+			"fieldname":"payment_detail",
+			"label": __("Payment Detail?"),
```

Comment:
We generally avoid this type of label with question mark. Change it to "Show Payment Details"? Same in case of "POS?", it can be "Show only POS"